db/mysql/db.go

```diff
@@ -82,6 +84,9 @@ func (c mysqlCreator) Create(p *properties.Properties) (ycsb.DB, error) {
 	db.SetMaxOpenConns(threadCount * 2)
 
 	d.verbose = p.GetBool(prop.Verbose, prop.VerboseDefault)
+	if p.GetBool(mysqlForceIndex, false) {
```

Comment:
maybe default is true is better? so we can test workload e directly.

Reply:
ok!